bigtable-dataflow-parent/bigtable-beam-import/README.md

```diff
@@ -168,15 +169,16 @@ Please pay attention to the Cluster CPU usage and adjust the number of Dataflow
     ```
 1. Run the import.
     ```
-    java -jar bigtable-beam-import-2.0.0-alpha1.jar import \
+    java -jar bigtable-beam-import-2.0.0.jar import \
         --runner=dataflow \
         --project=$PROJECT_ID \
-        --bigtableInstanceId=$INSTANCE_D \
+        --bigtableInstanceId=$INSTANCE_ID \
         --bigtableTableId=$TABLE_NAME \
-        --sourcePattern='$BUCKET_NAME/hbase-export/part-*' \
```

Comment:
Have you run through the commands to make sure it works ? Billy updated the read me with the commands he ran.

Reply:
yes I just tried it, I think with the quotes it'll try to find `$BUCKET_NAME/...` instead of using the $BUCKET_NAME environment variable